library/src/config/config.ts

```diff
@@ -1,12 +1,11 @@
-import { Options as ParserOptions } from 'json-schema-ref-parser';
 import { PushStateBehavior } from '../types';
 
 export interface ConfigInterface {
   schemaID?: string;
   show?: ShowConfig;
   expand?: ExpandConfig;
   showErrors?: boolean;
-  parserOptions?: ParserOptions;
+  parserOptions?: any;
```

Comment:
any ❤️

Reply:
I checked that parser-js has different signature than ParserOptions from `json-schema-parser`